### PR TITLE
Incorrect number of behind messages in persistent subscriptions dashboard [DB-116]

### PR DESCRIPTION
**Pull request by dhingrak**
Fixed: Incorrect number of behind messages in persistent subscrptions dashboard.

Fixes https://github.com/EventStore/EventStore.UI/issues/336

Currently # of messages in persistent subscription dashboard is showing 1 / 0 (behind messages) even the stream does not exists. The goal of this PR to fix the incorrect number of behind messages.

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -1,53 +1,57 @@
 @font-face {
-  font-family: 'Roboto';
+  font-family: "Roboto";
```
Comment (hayley-jean) [inline src/css/main.css:2]: It looks like this file was auto-formatted. Can you undo these changes as they are not relevant to the fix

**Review by hayley-jean — COMMENTED** [review]
I can't reproduce the issue in 22.10.2. Was the off-by-one error still showing in the UI before this change?

**Comment by hayley-jean** [comment]
> I can't reproduce the issue in 22.10.2. Was the off-by-one error still showing in the UI before this change?

The off-by-one error does show in the latest UI, but EventStore.ClusterNode.Web hasn't been updated in a while so the issue isn't present when building EventStore from master.

Once this PR is merged, we should update the UI in EventStore master as well

**Review by hayley-jean — APPROVED** [review]

**Review by timothycoleman — APPROVED** [review]
